onnx/numpy_helper.py

```diff
@@ -12,7 +12,7 @@
 from six import text_type, binary_type
 from typing import Sequence, Any, Optional, Text, List
 
-if platform.system() != 'AIX' and sys.byteorder != 'little':
+if (platform.system() != 'AIX' and platform.processor() != 's390x') and sys.byteorder != 'little':
```

Comment:
If we did add the `byteswap` workaround to `numpy_helper.to_array`, would we need this increasingly complex condition here?

This condition is growing complex. It would be a good idea to add a small, well named helper function such as `_get_system_byteorder_supported()` and move the complexity there.

Reply:
I removed this condition check because we can say "numpy helper is available on big-endian system" by adding endian conversion in numpy helper.